README.md

```diff
@@ -136,51 +144,73 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
          "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
        }
        ```
-3. Try getting scan report (in unified Harbor format understandable by Harbor Web console):
+3. Try getting scan report (in the new v1.1 format understandable by Harbor Web console):
    ```
-   curl -H 'Accept: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' \
+   curl -H 'Accept: application/vnd.security.vulnerability.report; version=1.1' \
      http://scanner-adapter:8080/api/v1/scan/3fa85f64-5717-4562-b3fc-2c963f66afa6/report
 
    Retry-After: 15
    Status: 302 Found
    ```
 4. Wait 15 seconds or use your own retry interval ...
 5. ... and try getting scan report again:
+   
    ```
-   curl -H 'Accept: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' \
-     http://scanner-adapter:8080/api/v1/scan/3fa85f64-5717-4562-b3fc-2c963f66afa6/report
-
-   Content-Type: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0
-   Status: 200 OK
-
-   {
-     "generated_at": "2019-08-07T12:17:21.854Z",
-     "artifact": {
-       "repository": "library/mongo",
-       "digest": "sha256:917f5b7f4bef1b35ee90f03033f33a81002511c1e0767fd44276d4bd9cd2fa8e"
-     },
-     "scanner": {
-       "name": "Microscanner",
-       "vendor": "Aqua Security",
-       "version": "3.0.5",
-     },
-     "severity": "High",
-     "vulnerabilities": [
-       {
-         "id": "CVE-2017-8283",
-         "package": "dpkg",
-         "version": "1.17.27",
-         "fix_version": "1.18.0",
-         "severity": "High",
-         "description": "...",
-         "links": [
-           "https://security-tracker.debian.org/tracker/CVE-2017-8283"
-         ]
-       },
-       ...
-     ]
-   }
-   ```
+   "vulnerabilities": [
+    {
+      "id": "CVE-2020-1967",
+      "package": "libcrypto1.1",
+      "version": "1.1.1c-r0",
+      "fix_version": "1.1.1g-r0",
+      "severity": "High",
+      "description": "Server or client applications that call the SSL_check_chain() function during or after a TLS 1.3 handshake may crash due to a NULL pointer dereference as a result of incorrect handling of the \"signature_algorithms_cert\" TLS extension. The crash occurs if an invalid or unrecognised signature algorithm is received from the peer. This could be exploited by a malicious peer in a Denial of Service attack. OpenSSL version 1.1.1d, 1.1.1e, and 1.1.1f are affected by this issue. This issue did not affect OpenSSL versions prior to 1.1.1d. Fixed in OpenSSL 1.1.1g (Affected 1.1.1d-1.1.1f).",
+      "links": [
+        "https://avd.aquasec.com/nvd/cve-2020-1967"
+      ],
+      "layer": {
+        "digest": "sha256:9d48c3bd43c520dc2784e868a780e976b207cbf493eaff8c6596eb871cbd9609",
+        "diff_id": "sha256:03901b4a2ea88eeaad62dbe59b072b28b6efa00491962b8741081c5df50c65e0"
+      },
```

Comment:
This is specific to Trivy. It's not specified by the API. Let's remove those extra properties

Reply:
Done. My bad - I copied the sample response snippet, but forgot to clean-up this property